distributed/nanny.py

```diff
@@ -456,7 +463,7 @@ async def plugin_add(self, plugin=None, name=None):
                     result = await result
             except Exception as e:
                 msg = error_message(e)
-                return msg
+                return cast(dict[str, Any], msg)
```

Comment:
Maybe you could you change the return type of the function to ErrorMessage instead?